README.md

```diff
@@ -1,6 +1,5 @@
 # Kafka Connect Elasticsearch Connector
-[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fconfluentinc%2Fkafka-connect-elasticsearch.svg?type=shield)](https://app.fossa.io/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fconfluentinc%2Fkafka-connect-elasticsearch?ref=badge_shield)
-
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fconfluentinc%2Fkafka-connect-elasticsearch.svg?type=shield)](https://app.fossa.io/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fconfluentinc%2Fkafka-connect-elasticsearch?ref=badge_shield) [Changelog](https://docs.confluent.io/kafka-connect-elasticsearch/current/changelog.html)
```

Comment:
@vdm this link looks odd in the screenshot, could we please add Changelog on a new line? We can add something like 

Changelog for this connector can be found [here](https://docs.confluent.io/kafka-connect-elasticsearch/current/changelog.html).



![image](https://user-images.githubusercontent.com/29556518/147113644-ca44e236-0dba-4660-b9ed-e53121bfd027.png)